apps/inheritance_app/inheritance_encrypt_data.c

```diff
@@ -673,6 +674,7 @@ static bool encrypt_data(void) {
       status = false;
       break;
     }
+    set_app_flow_status(INHERITANCE_ENCRYPT_DATA_STATUS_MESSAGE_ENCRYPTED);
```

Comment:
This would false update cysync even if msg encrpytion fails.
You can put it inside encrypt_message when it succeeds

Reply:
Flow status is idependent of erros